<h1 style='font-size:40px'> 
    <a href='https://otexts.com/fpp2/residuals.html'>Residuals</a></h1>
<div> 
    <ul style='font-size:20px'> 
        <li> 
        </li>
    </ul>
 </div>

In [4]:
! git add .
! git commit -am 'Mencionar critérios de avaliação de resíduos para modelos'
! git push

[master 01479f6] Continuar livro-site da seção "3.3 Residual diagnostics"
 1 file changed, 13 insertions(+), 13 deletions(-)
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 24 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 599 bytes | 599.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
To https://github.com/felipesveiga/time-series.git
   4ed3dd6..01479f6  master -> master


<p style='color:red'> Iniciei "3.3 Residual diagnostics"; Mencionar critérios de avaliação de resíduos para modelos